policybot/pkg/storage/store.go

```diff
@@ -16,8 +16,13 @@ package storage
 
 import (
 	"io"
+
+	google_spanner "cloud.google.com/go/spanner"
 )
 
+// IssueIterator itertates the spanner query returned working set.
+type IssueIterator func(row *google_spanner.Row) error
```

Comment:
I don't think we need this anymore, right?

Reply:
removed.